### PR TITLE
fix: unorphaned `avm/res/databricks/access-connector`

### LINKED ISSUE #961: [Failed pipeline] avm.res.databricks.access-connector
Failed run: https://github.com/Azure/bicep-registry-modules/actions/runs/7811600299

### PR DESCRIPTION
**Pull request by krbar**
## Description

Unorphaned `avm/res/databricks/access-connector`

### CODE REVIEW

File: avm/res/databricks/access-connector/main.json [hunk 1]
```diff
@@ -5,8 +5,8 @@
   "metadata": {
     "_generator": {
       "name": "bicep",
-      "version": "0.24.24.22086",
-      "templateHash": "5801242046314845466"
+      "version": "0.25.3.34343",
```
Comment (eriqua) [inline avm/res/databricks/access-connector/main.json:8]: This would cause a new version to be published. We are saving that to publish all modules with the new `--with-source` option. Could you please manually revert this change?

Comment (krbar) [reply avm/res/databricks/access-connector/main.json]: Done

Comment (eriqua) [reply avm/res/databricks/access-connector/main.json]: @clintgrove thanks for owning this module. Could you please review & approve this PR?

**Review by clintgrove — APPROVED** [review]
Looks like MD files have only changed and other versions on main.json have been reverted
